src/renderer.rs

```diff
@@ -259,6 +275,25 @@ fn css() -> Markup {
         color: #3498db;
         text-decoration: none;
     }
+    .download {
+        display: flex;
```

Comment:
I kind of don't like how this pushes the rest of the content away quite a bit. How about instead position the download button right of the header where it tells you the current directory?

Reply:
You're right. I did some experiments and it's kind of hard to make it look right with very long path though. How about just putting on a new line, but align to the left ? Here's a screenshot, with 1 and then several download buttons

<img width="1335" alt="Screenshot 2019-03-15 at 17 22 21" src="https://user-images.githubusercontent.com/47121073/54446685-ced7a580-4747-11e9-9cf1-eb48790da534.png">

<img width="1340" alt="Screenshot 2019-03-15 at 17 23 02" src="https://user-images.githubusercontent.com/47121073/54446659-c1222000-4747-11e9-9ffc-2be94f838840.png">